docs/src/main/sphinx/connector/jdbc-common-configurations.fragment

```diff
@@ -51,4 +51,11 @@ connector:
     JDBC query. Using a large timeout can potentially result in more detailed
     dynamic filters. However, it can also increase latency for some queries.
     Defaults to `20s`.
+* - `statistics.enabled`
+  - Populate table statistics and use the [cost-based
+    optimizer](/optimizer/cost-based-optimizations) to improve query
+    performance. Defaults to `true`, except for the [Vertica
```

Comment:
Lets not do this .. instead add a property in the header and set it accordingly for all connectors. And then also rejig Table statistics in the performance section to be used in all connector pages affected ..